src/schema/frugalos.rs

```diff
@@ -260,3 +271,19 @@ impl Call for TakeSnapshotRpc {
     type ResDecoder = BincodeDecoder<Self::Res>;
     type ResEncoder = BincodeEncoder<Self::Res>;
 }
+
+/// A RPC for repairing objects.
+#[derive(Debug)]
+pub struct RepairObjectRpc;
+impl Call for RepairObjectRpc {
+    const ID: ProcedureId = ProcedureId(0x000a_0002);
+    const NAME: &'static str = "frugalos.ctrl.repair_object";
```

Comment:
I’ve been on the fence but I decided to include `RepairObjectRpc` into `frugalos.ctrl` namespace.